.vscode/launch.json

```
@@ -0,0 +1,22 @@
+{
+  "configurations": [
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Jest: current file",
+      "program": "${workspaceFolder}/node_modules/.bin/jest",
+      "args": [
+        "${fileBasenameNoExtension}",
+        "--config",
+        "--runInBand",
+        "jest.config.js"
```

Is `jest.config.js` meant to be passed to `--config`?

```suggestion
        "--config",
        "jest.config.js",
        "--runInBand"
```